frontend/src/Components/game.js

```diff
@@ -91,23 +102,53 @@ class Game extends React.PureComponent {
       });
   };
 
+  redirectToGrid = () => {
+    if (this.props.location.pathname !== '/save' && this.props.location.pathname !== '/'){
```

Comment:
This is essentially equivalent to if the route is /manageSaves, but a more generic form in case that another route may be added in the future.